content/applications/inventory_and_mrp/manufacturing/management/configure_manufacturing_product.rst

```diff
@@ -0,0 +1,73 @@
+===================================
+Manufacturing product configuration
+===================================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+In order to manufacture a product in Odoo *Manufacturing*, the product must be properly configured.
+Doing so consists of enabling the Manufacturing route and configuring a Bill of Materials (BoM) for
+the product. Once these steps are completed, the product is selectable when creating a new
+manufacturing order.
+
+Activate the Manufacture route
+==============================
+
+The Manufacture route is activated for each product on its product page. To do so, begin by
+navigating to :menuselection:`Manufacturing --> Products --> Products`, then select an existing
+product, or create a new one by clicking :guilabel:`New`.
```

Comment:
Italicize the setting name.

The first sentence here, as it's written, confused me I had to read it twice.
I think it means to communicate that the route is activated on a product's _own/unique_ product page.

We're also missing a conjunction in the second sentence, and there are possibly some mixed tenses here.

Suggestion # 1, which retains most of what you wrote originally
```suggestion
The *Manufacture* route is activated for each product, on its own product page. Do so by
navigating to :menuselection:`Manufacturing --> Products --> Products`, and then selecting an existing
product, or create a new one by clicking :guilabel:`New`.
```

_
Suggestion # 2 condenses two sentences into one, and adds time order language to indicate that this is the first step.
```suggestion
First, activate the *Manufacture* route on a product's page by navigating to
:menuselection:`Manufacturing --> Products --> Products`, and then select an existing
product (or create a new one by clicking :guilabel:`New`).
```